pkgs/top-level/all-packages.nix

```diff
@@ -20003,7 +20003,7 @@ in
   check_systemd = callPackage ../servers/monitoring/nagios/plugins/check_systemd.nix { };
 
   neo4j = callPackage ../servers/nosql/neo4j {
-    jre = jre8_headless;
+    jre = openjdk11;
```

Comment:
I'm not exactly sure if neo4j runs properly with headless or not.
If it can, using openjdk11_headless would reduce the size of the resulting closure.
If we are not sure, I would prefer to test with openjdk11_headless

Reply:
I tried running a few things using the headless and it seems to work fine so made the switch.